google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java

```diff
@@ -109,6 +115,9 @@ public void hyperlinkUpdate(@NotNull Notification notification, @NotNull Hyperli
       String href = event.getDescription();
 
       if (DEACTIVATE_LINK_HREF.equals(href)) {
+        UsageTrackerProvider.getInstance()
+            .trackEvent(GctTracking.APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK)
```

Comment:
whats the difference between APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK and APP_ENGINE_OLD_PLUGIN_DEACTIVATED ?